lua/tardis/sh_screens.lua

```diff
@@ -262,50 +276,181 @@ concommand.Add("tardis_toggleui", function()
+		titlebar:SetPos(0, frame:GetTall() - titlebar:GetTall() )
+		titlebar:SetBackgroundColor(Color(1, 1, 100, 255))
+	else
+		titlebar:SetSize(frame:GetWide(), frame:GetTall() * 0.1)
+		titlebar:SetPos(0, 0)
+		titlebar:SetBackgroundColor(Color(220,220,220))
+	end
+
 	local pagename = vgui.Create("DLabel",titlebar)
-	pagename:SetTextColor(Color(0,0,0))
+	if TARDIS:GetSetting("visual_gui_enabled")
+	then
+		pagename:SetTextColor(Color(255,255,255))
+	else
+		pagename:SetTextColor(Color(0,0,0))
+	end
 	pagename:SetFont("TARDIS-PageName")
 	pagename.DoLayout = function(self)
 		pagename:SizeToContents()
-		pagename:SetPos((titlebar:GetWide()*0.5)-(pagename:GetWide()*0.5),(titlebar:GetTall()*0.5)-(pagename:GetTall()*0.5))
+		pagename:SetPos((titlebar:GetWide()*0.5)-(pagename:GetWide()*0.5),
+						(titlebar:GetTall()*0.5)-(pagename:GetTall()*0.5))
 	end
 	pagename:SetText("")
 	pagename:DoLayout()
 	screen.pagename=pagename
-	
-	local backbutton = vgui.Create("DButton",titlebar)
-	backbutton:SetVisible(false)
+
+	local main = vgui.Create("DPanel",frame)
+	main:SetSize(frame:GetWide(),frame:GetTall() - titlebar:GetTall())
+	if TARDIS:GetSetting("visual_gui_enabled")
+	then
+		main:SetPos(0,0)
+		main:SetBackgroundColor(Color(0,0,0,255))
+	else
+		main:SetPos(0, titlebar:GetTall())
+	end
+	screen.main=main
+
+	local mmenu = vgui.Create("DPanel",main)
+	mmenu:SetSize(main:GetSize())
+	mmenu:SetPos(0,0)
+	mmenu:SetBackgroundColor(Color(0,0,0,255))
+	local mmenu_background=vgui.Create("DImage", mmenu)
+	mmenu_background:SetImage(background_img)
+	mmenu_background:SetSize( mmenu:GetWide(), mmenu:GetTall() )
+	screen.mmenu=mmenu
+
+	local backbutton, menubutton, exitpopup_button
+
+	if TARDIS:GetSetting("visual_gui_enabled")
+	then
+		titlebar.button_size = math.min(titlebar:GetTall() * 0.8, titlebar:GetWide() * 0.25)
+		titlebar.button_posY = titlebar:GetTall() * 0.5 - titlebar.button_size * 0.5
+
+		menubutton = TardisScreenButton:new(titlebar)
+		menubutton:SetSize(titlebar.button_size * 2, titlebar.button_size)
+		menubutton:SetPos(0, titlebar.button_posY)
+		menubutton:SetIsToggle(false)
+
+		backbutton = TardisScreenButton:new(titlebar)
+		backbutton:SetSize(titlebar.button_size * 2, titlebar.button_size)
+		backbutton:SetPos(titlebar:GetWide() * 0.3 - titlebar.button_size, titlebar.button_posY)
+		backbutton:SetIsToggle(false)
+
+		exitpopup_button = TardisScreenButton:new(titlebar)
+		exitpopup_button:SetSize(titlebar.button_size * 2, titlebar.button_size)
+		exitpopup_button:SetPos(titlebar:GetWide() * 0.95 - titlebar.button_size, titlebar.button_posY)
+		exitpopup_button:SetIsToggle(false)
+
+		local left_arrow, right_arrow
+
+		local left_arrow = TardisScreenButton:new(titlebar)
+		left_arrow:SetSize(titlebar.button_size * 2, titlebar.button_size)
+		left_arrow:SetPos(titlebar:GetWide() * 0.3 - titlebar.button_size, titlebar.button_posY)
+		left_arrow:SetIsToggle(false)
+		left_arrow:SetText("left")
```

Comment:
Fallback text for left / right arrows should be < and >

Reply:
Replied in Discord

Reply:
Happy for this to be left as part of making textonly hidden as part of another comment